### PR TITLE
[Depends on #1441][jsk_apc2016_common] add publish_target_bin

### PR DESCRIPTION
**Pull request by yuyu2172**
This node publishes a BinInfo of a target bin.
It is nice to use the published BinInfo because nodes that use it do not need to do condition handling of target_bin_name.

### CODE REVIEW

**Comment by knorth55** [comment]
publish_target_bin_info is better, i think

**Comment by yuyu2172** [comment]
right

File: jsk_apc2016_common/node_scripts/publish_target_bin_info.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            continue
+        bin_info_dict[target_bin_name].header.seq = (
+                bin_info_dict[target_bin_name].header.seq + 1)
+        target_bin_info = bin_info_dict[target_bin_name]
+        target_bin_info.header.stamp = rospy.Time.now()
+        target_bin_info_pub.publish(target_bin_info)
+
+
+if __name__ == '__main__':
+    rospy.init_node('sib_publish_target_bin_info')
```
Comment (knorth55) [inline jsk_apc2016_common/node_scripts/publish_target_bin_info.py]: `publish_target_bin_info` is better

Comment (yuyu2172) [inline jsk_apc2016_common/node_scripts/publish_target_bin_info.py]: why?

**Comment by yuyu2172** [comment]
a new commit in this PR is 'add publish target_bin'.

**Comment by knorth55** [comment]
i think nodes should be in `jsk_2016_01_baxter_apc` package, not in `jsk_apc2016_common` like #1426

**Comment by wkentaro** [comment]
Maybe it should be installed in `jsk_apc2016_common`. As discussion in
https://github.com/start-jsk/jsk_apc/pull/1332#issuecomment-214097253

**Comment by knorth55** [comment]
i don't understand the difference of these packages.
is these common nodes?

**Comment by knorth55** [comment]
+1

**Pull request merged**